middleware/src/main.rs

```diff
@@ -0,0 +1,29 @@
+extern crate actix;
+extern crate actix_web;
+
+use actix_web::{server, App};
+
+#[allow(dead_code)]
+mod redirect;
+#[allow(dead_code)]
+mod simple;
+
+fn main() {
+    let sys = actix::System::new("middleware-example");
+
+    let _addr = server::new(|| {
+        App::new()
+            .middleware(simple::SayHi)
+            // .middleware(redirect::CheckLogin)
```

Comment:
could you enable this middleware or remove it?

Reply:
I can, of course (which ever you prefer); but I wanted to isolate each middleware and not have them run on top of each other as that only makes it more difficult to understand what is happening.

I don't mind removing it either, but then the example would not have been helpful to myself (as a  very new to Rust dev) a week or two ago as I could not figure out how to conditionally halt a request.